src/components/ae-card/ae-card.md

```diff
@@ -38,11 +58,31 @@
     <ae-text fill="white" contenteditable>My Account</ae-text>
   </template>
   <template slot="header">
-    <ae-text fill="white" face="mono-base">20.65 AE</ae-text>
+    <ae-dropdown>
+      <ae-icon name="more" fill="white" size="20px" slot="button" />
+      <li>
+        <ae-button>
+          <ae-icon name="copy" />
+          Copy Address
+        </ae-button>
+      </li>
+      <li>
+        <ae-button>
+          <ae-icon name="share" />
+          Share Address
+        </ae-button>
+      </li>
+      <li>
+        <ae-button>
+          <ae-icon name="eye" />
+          Details
+        </ae-button>
+      </li>
+    </ae-dropdown>
   </template>
   <ae-text face="uppercase-xs" weight=600 style="margin: 0">Normal Secured</ae-text>
   <ae-address value="ak$QY8VNEkhj7omMUjAvfVBq2NjTDy895LBYbk7qVxQo1qT8VqfE" length="medium" gap=0 />
-  <ae-toolbar fill="primary" align="right" slot="footer">
+  <ae-toolbar fill="primary" fill="white" align="right" slot="footer">
```

Comment:
It looks a little bit strange:
![screenshot from 2018-10-09 19-17-13](https://user-images.githubusercontent.com/9007851/46686233-f4d8a700-cbf7-11e8-9796-847cb4add1a7.png)

Reply:
Yep, my mistake.